src/llvm-coverage-instrumentation.md

```diff
@@ -111,7 +112,7 @@ fn some_func(flag: bool) {
 In this example, four contiguous code regions are counted while only
 incrementing two counters.
 
-CFG analysis is used to not only determine *where* the branches are, for
+CFG analysis is used to not only determine _where_ the branches are, for
```

Comment:
Why change this?

Reply:
Not intended, as explained above. Sorry.

Reply:
Leaving this one as-is, to be internally consistent (see my other comment explaining what I kept and why).